optima/programs.py

```diff
@@ -193,15 +193,15 @@ def progs_by_targetpar(self, filter_partype=None):
         else: return progs_by_targetpar
 
 
-    def getdefaultbudget(self, years=None, verbose=2):
+    def getdefaultbudget(self, t=None, verbose=2):
         ''' Extract the budget if cost data has been provided'''
         
         # Initialise outputs
         totalbudget, lastbudget, selectbudget = odict(), odict(), odict()
 
         # Validate inputs
-        if type(years) in [int, float]: years = [years]
-        if isinstance(years,ndarray): years = years.tolist()
+        if type(t) in [int, float]: t = [t]
```

Comment:
please use isnumber() from utils, because if e.g. `t` is type `numpy.float64` then this will fail